.config-schema.json

```diff
@@ -451,6 +451,12 @@
                     "type": "integer",
                     "default": "100",
                     "x-env-variable": "OPENFGA_DISPATCH_THROTTLING_THRESHOLD"
+                },
+                "maxThreshold": {
+                    "description": "define the maximum dispatch threshold beyond above which requests will be throttled. 0 will use the threshold value as maximum",
+                    "type": "integer",
+                    "default": "0",
```

Comment:
It appears the default max is 200 in the code. Why is it 0 here?

Reply:
0 means using the threshold as maximum.  The default max in code is 0